### ISSUE TITLE
ContextReTimingOfLandings

### ISSUE THREAD

**Issue by rsryman** [rryman/BeneshForBalletBook3#46]
We should explain our decision to simplify the notation of timing for the final landing in allegro enchaînements. Let's add a note when it first appears in the Allegro Enchaînement chapter. Suggestion:

The final landing and re-stretch is shown to occur/notated on Bar ?? count?? It is generally understood that the dancer finishes an allegro enchaînement with a well-cushioned plié, then re-stretches as needed to regain stability. This timing varies in context, according to the tempo or quality, the dancer, and the school.

**Comment by rryman** [comment]
I don't know how to incorporate this. It is a lot of text to put in the "i" box in each case that it is needed. Any suggestions? Google Hangout on this?

**Comment by rsryman** [comment]
Perhaps just add a note in your TODO issue, then we can decide when we get there. Maybe reword slightly and add to the section page for Allegro Enchaînements?

**Comment by rsryman** [comment]
Good.
